include/matrix.h

```diff
@@ -129,7 +129,7 @@ class Matrix : public MatrixExpression<Matrix<TDataType, TSize1, TSize2>,
     }
 
     void resize(std::size_t NewSize1, std::size_t NewSize2) {
-        base_type::resize(NewSize1, NewSize2);
+        //base_type::resize(NewSize1, NewSize2);
```

Comment:
@msandre why this one is commented?

Reply:
Sorry but I don't remember why I commented this at the time. I think it can be ignored.